recipe/meta.yaml

```diff
@@ -285,7 +286,7 @@ outputs:
         - python
         - {{ pin_compatible('numpy') }}
         - {{ pin_subpackage("libboost-python", exact=True) }}
-        - {{ pin_subpackage("libboost-headers", exact=True) }}
+        - {{ pin_subpackage("libboost-devel", exact=True) }}
```

Comment:
@isuruf, I'm almost certain that run-depending in `libboost-devel` ensures that anyone host-depending on `libboost-python-devel` will in fact pick up the `libboost` run-export. If you want me to make that more explicit, I'm happy to do that.

Reply:
Not true.

Reply:
Huh, TIL - I though it picks all the run-exports from what's present in host (after resolution). In any case, I added the explicit run-export on `libboost` now.